pages/desktop/user.py

```diff
@@ -117,10 +119,27 @@ def notification_header_text(self):
 
     def click_update_account(self):
         self.selenium.find_element(*self._update_account_locator).click()
-        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.update_message == "Profile Updated")
+        WebDriverWait(self.selenium, self.timeout).until(
+            lambda s: self.update_message == "Profile Updated")
+
+    def hide_email_address(self):
+        email_checkbox = self.selenium.find_element(*self._hide_email_checkbox_locator)
+        WebDriverWait(self.selenium, self.timeout).until(
+            expected_conditions.visibility_of(email_checkbox))
+
+        if email_checkbox.is_selected():
+            raise Exception('Email is already hidden')
+            self.selenium.find_element(*self._hide_email_checkbox_locator).click()
```

Comment:
Note that this code would never be reached, as it's within an `if` statement that raises an exception immediately before.